test/callbacks/cases_matching_callbacks/9.btx_model.yaml

```diff
@@ -0,0 +1,9 @@
+:environment:
+- :name: entry_1
+  :type: string
+- :name: entry_2
+  :type: integer_signed
+:stream_classes:
+- :name: sc
+  :event_classes:
+  - :name: event
```

Comment:
Where is the `outpub` to verify?
We should just use `text-pretty` with the hostname,
or test-detail your call

Reply:
It depends on what we want to verify. If we want to verify the value, and assert should be enough, right?
if we want to verify it is in the enviroment `details` should work. On the other hand, for some reason, `pretty` is not displaying the environemnt entries, i.e., it is not doing `env1 event: {...}`.

Reply:
You are right, but I think for this test we can not use `details` since we need to test that matching callbacks received the value properly (we do this in the filter callbacks). If we use details, it we will display 2 traces, one of them will have the enviroment set and the other don't. So the later will let us ensure the enviorment has been set properly, but the first test do that since we are extracting the data from the enviroment. I can take a look on that again in monday.